compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -1546,11 +1546,39 @@ FailureOr<TilingResult> WinogradOutputTransformOp::getTiledImplementation(
   outputOffsets[hDim] = hSizeAndOffset.second;
   outputOffsets[wDim] = wSizeAndOffset.second;
 
+  Value outputSlice = getSlice(builder, loc, getOutput(), outputOffsets,
+                               outputSizes, outputStrides);
+  // The image dims of the winograd.output_transform result will always be a
+  // multiple of the static output_tile_size, so insert a tensor.cast op to
+  // maintain more static information in the IR.
+  auto outSliceType = cast<ShapedType>(outputSlice.getType());
+  SmallVector<int64_t> staticOutShape = llvm::map_to_vector(
+      llvm::seq<int64_t>(outSliceType.getRank()),
+      [&](auto idx) { return outSliceType.getDimSize(idx); });
```

Comment:
I think this is just a `outSliceType.getShape()`? So something like `SmallVector<int64_t> staticOutShape(outSliceType.getShape())`;